### DISCUSSION TITLE
API Proxmox Ticket

### DISCUSSION THREAD

**Discussion by darvad** [emersonfelipesp/netbox-proxbox#80] [Q&A]
Hi. I am new to Netbox and API. Trying to get VM info from my Proxmox HV into Netbox. This plugin seems perfect. But cannot get it to work. I followed the guide but always get following error when trying hitting Proxmox Full Update  button in the netbox plugin section. Tried it with root user also. Same error
<class 'proxmoxer.core.ResourceException'>
401 Unauthorized: no such token 'tokenID' for user 'api@pve' - {'errors': b''}
Any help is welcome.
Thanks ahead
Steve

**Comment by emersonfelipesp** [comment]
Could you please send me your `PLUGINS` and `PLUGINS_CONFIG` variables? Don't know if you are still trying to use it, but would like to help you!

**Reply by darvad** [reply]
Thanks for reply. But in the meantime I have switched to use a python script for the task.

**Reply by emersonfelipesp** [reply]
> Thanks for reply. But in the meantime I have switched to use a python script for the task.

That's right! I'm going to close this discussion, feel free to come later if you wish to test it again.
